zuul-core/src/main/java/com/netflix/zuul/message/http/Cookies.java

```diff
@@ -30,6 +37,23 @@
  */
 public class Cookies
 {
+    private static final Logger LOG = LoggerFactory.getLogger(Cookies.class);
+
+    private static final CachedDynamicBooleanProperty CLEAN_COOKIES = new CachedDynamicBooleanProperty(
+            "zuul.HttpRequestMessage.cookies.clean", false);
+
+    private static final List<Pattern> RE_STRIP;
+    /** ":::"-delimited list of regexes to strip out of the cookie headers. */
+    private static final DynamicStringProperty REGEX_PTNS_TO_STRIP_PROP =
```

Comment:
Long term I think this needs to be removed, because it is a hack in the first place.    I think it also doesn't work with the other values like `Expires` which sometimes get through.

Can you mark this `@Deprecated` with a comment on the intent to remove?

Reply:
Done.